Translations/translation_JA_JP.json

```diff
@@ -7,22 +7,22 @@
     "SettingsCalibrationDone": "校正完了",
     "SettingsCalibrationWarning": "続行する前に、コテ先が室温と同じになっていることを確認してください",
     "SettingsResetWarning": "設定をリセットしますか？",
-    "UVLOWarningString": "電圧が低すぎます",
-    "UndervoltageString": "Undervoltage",
-    "InputVoltageString": "Input V: ",
-    "WarningTipTempString": "Tip temp: ",
-    "BadTipString": "BAD TIP",
+    "UVLOWarningString": "DC電圧が低すぎます",
+    "UndervoltageString": "電圧不足",
+    "InputVoltageString": "入力電圧 V: ",
+    "WarningTipTempString": "コテ先温度: ",
+    "BadTipString": "コテ先エラー",
     "SleepingSimpleString": "Zzzz",
-    "SleepingAdvancedString": "Sleeping...",
-    "WarningSimpleString": "HOT!",
-    "WarningAdvancedString": "!!! HOT TIP !!!",
-    "SleepingTipAdvancedString": "Tip:",
-    "IdleTipString": "Tip:",
-    "IdleSetString": " Set:",
-    "TipDisconnectedString": "NO TIP",
-    "SolderingAdvancedPowerPrompt": "Power: ",
+    "SleepingAdvancedString": "スリープ中...",
+    "WarningSimpleString": "高温注意!",
+    "WarningAdvancedString": "!!! コテ先高温 !!!",
+    "SleepingTipAdvancedString": "コテ先:",
+    "IdleTipString": "コテ先:",
+    "IdleSetString": " 設定:",
+    "TipDisconnectedString": "コテ先が接続されていません",
+    "SolderingAdvancedPowerPrompt": "電源: ",
```

Comment:
Please mind that most of strings are for the advanced standby/soldering screens and uses the small font, in which CJK cannot be displayed.